.tekton/discovery-server-pull-request.yaml

```diff
@@ -452,71 +493,6 @@ spec:
         operator: in
         values:
         - "false"
-    ## TODO uncomment sast-coverity-check after KONFLUX-6806 is fixed
```

Comment:
per code added above, it seems `sast-coverity-check` is now working. we might need this particular code change, if that's true